modules/nomad-acl/README.md

```diff
@@ -0,0 +1,96 @@
+# Nomad ACL
+
+[ACL](https://www.nomadproject.io/guides/acl.html) can be enabled for Nomad so that only users
+with the necessary tokens can submit jobs. This module only enables to built in access controls
+provided by the ACL facility in the Open Source version of Nomad. Additional controls provided
+by Sentinel in the Enterprise version is not enabled.
+
+This module __does not__ create the necessary Vault and Nomad policies. These policies are very
+specific to your use case and you should define them yourselves.
+
+## Integration with `Core` module
+
+This module is integrated with the `core` module to enable you to use both in conjunction
+seamlessly.
+
+## Pre-requisites
+
+You must have Terraformed the [core](../core) module first. In addition, you must have at least
+initialised and unsealed the Vault servers.
+
+Make sure you have properly configured Vault with the appropriate
+[authentication methods](https://www.vaultproject.io/docs/auth/index.html) so that your users can
+authenticate with Vault to get the necessary tokens and credentials.
+
+### Bootstrapping Nomad ACL
+
+We have to manually [bootstrap](https://www.nomadproject.io/guides/acl.html#bootstrapping-acls) ACLs
+on the Nomad servers first.
+
+Firstly, this requires that the Nomad servers and servers are configured with the
```

Comment:
Suggestion: `this requires that the Nomad servers and servers are configured with the` -> `this requires the Nomad clients and servers to be configured with`.